src/rgw/rgw_rest_pubsub.cc

```diff
@@ -423,6 +423,48 @@ auto find_unique_topic(const rgw_pubsub_bucket_topics& bucket_topics, const std:
 }
 }
 
+void remove_notification_by_topic(const DoutPrefixProvider *dpp, const std::string& topic_name, const RGWPubSub::BucketRef& b, optional_yield y) {
```

Comment:
if this is a standalone function you should return "int" and not "void"
setting "op_ret" inside the function has no effect